timesketch/api/v1/resources.py

```diff
@@ -339,11 +342,25 @@ def post(self, sketch_id):
             for timeline in sketch.timelines:
                 tl_colors[timeline.searchindex.index_name] = timeline.color
                 tl_names[timeline.searchindex.index_name] = timeline.name
+
+            histogram = None
+            es_total_count_unfiltered = 0
+            try:
+                buckets = result[u'aggregations'][u'data_type'][u'buckets']
```

Comment:
I agree, I will go for 2 and add accessor methods but this require a refactor of the result backend. This needs to be done anyways because I want the result object to be more than it is today. I'll do this in a later PR though as this involves some thinking before implementing.